src/openms/include/OpenMS/CHEMISTRY/DigestionEnzyme.h

```diff
@@ -53,6 +53,19 @@ namespace OpenMS
   {
   public:
 
+    /// Terminus Specificity
+    ///@todo Currently this is just here for encapsulation
```

Comment:
which I think is totally fine ;)

Reply:
but can't we just use the one from EnzymaticDigestion?

Reply:
yes, forgot to remove